components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -153,6 +162,17 @@ - (void)setSelectedItem:(UITabBarItem *)selectedItem {
   [self updateImageTintColorForAllViews];
 }
 
+- (void)setContentEdgeInsets:(UIEdgeInsets)contentEdgeInsets {
+  UIEdgeInsets adjustedInset = contentEdgeInsets;
+  if (self.mdf_effectiveUserInterfaceLayoutDirection == UIUserInterfaceLayoutDirectionRightToLeft) {
```

Comment:
I think this flipping should be done in an internal getter. Otherwise you have to respond to changes in the semantic content attribute and keep track of the original + flipped versions.